source/configuration_manual/sieve/usage.rst

```diff
@@ -0,0 +1,158 @@
+   the same vacation command
+
+-  The message contains at least one of the mailing list headers
+   "list-id", "list-owner", "list-subscribe", "list-post",
+   "list-unsubscribe", "list-help", or "list-archive"
+
+-  Auto-Submitted: header exists with any value except "no"
+
+-  Precedence: header exists with value "junk", "bulk" or "list"
+
+-  The envelope sender is considered a system address, which either:
+
+   -  begins with "MAILER-DAEMON" (case-insensitive),
+
+   -  begins with "LISTSERV" (case-insensitive),
+
+   -  begins with "majordomo" (case-insensitive),
+
+   -  contains the string "-request" anywhere within it
+      (case-sensitive), or
+
+   -  begins with "owner-" (case-sensitive)
+
+-  The envelope recipient and alternative addresses specified with the
+   vacation command's :addresses tag are not found in the message's To:,
+   Cc:, Bcc:, Resent-To:, Resent-Cc: or Resent-Bcc: fields.
+
+.. _sieve_usage-compiling_sieve_script:
+
+Manually Compiling Sieve Scripts
+--------------------------------
+
+When the Sieve plugin executes a script for the first time (or after it
+has been changed), it is compiled and stored in binary form (byte code)
+to avoid compiling the script again for each subsequent mail delivery.
+The Pigeonhole Sieve implementation uses the ``.svbin`` extension to
+store compiled Sieve scripts (e.g. ``.dovecot.svbin``). To store the
+binary, the plugin needs write access in the directory in which the
+script is located.
+
+A problem occurs when a global script is encountered by the plugin. For
+security reasons, global script directories are not supposed to be
+writable by the user. Therefore, the plugin cannot store the binary when
+the script is first compiled. Note that this doesn't mean that the old
+compiled version of the script is used when the binary cannot be
+written: it compiles and uses the current script version. The only real
+problem is that the plugin will not be able to update the binary on
+disk, meaning that the global script needs to be recompiled each time it
+needs to be executed, i.e. for every incoming message, which is
+inefficient.
+
+To mitigate this problem, the administrator must manually pre-compile
+global scripts using the ``sievec`` command line tool. For example:
+
+::
+
+   sievec /var/lib/dovecot/sieve/global/
+
+This is necessary for scripts listed in the :ref:`plugin-sieve-setting-sieve_global_path`,
+:ref:`plugin-sieve-setting-sieve_before` and :ref:`plugin-sieve-setting-sieve_after` settings.
+For global scripts that are only included in other scripts using the Sieve include extension,
+this step is not necessary, since included scripts are incorporated into
+the binary produced for the main script.
+
+Compile and Runtime Logging
+---------------------------
+
+Log messages produced during script compilation or during script
+execution are written to two locations by the LDA Sieve plugin:
+
+-  A log file is written in the same directory as the user's main
```

Comment:
The sieve_log setting overrides the log file name and it should be mentioned and referenced here.